packages/joint-core/types/joint.d.ts

```diff
@@ -913,18 +913,18 @@ export namespace dia {
         }
     }
 
-    class ElementView extends CellViewGeneric<Element> {
+    class ElementView<E extends Element = Element> extends CellViewGeneric<E> {
 
-        update(element?: Element, renderingOnlyAttrs?: { [key: string]: any }): void;
+        update(element?: E, renderingOnlyAttrs?: { [key: string]: any }): void;
 
         setInteractivity(value: boolean | ElementView.InteractivityOptions): void;
 
-        getDelegatedView(): ElementView | null;
+        getDelegatedView(): ElementView<E> | null;
```

Comment:
The `getDelegatedView()` function can return a different view (any of the ancestor views).

I think it should stay as it was here.
```ts
getDelegatedView(): ElementView | null;
```

Reply:
Makes sense, fixed